tests/cases/address_modes_float.amber

```diff
@@ -0,0 +1,123 @@
+#!amber
+# Copyright 2019 The Amber Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+SHADER vertex vert_shader PASSTHROUGH
+
+SHADER fragment frag_shader_red GLSL
+#version 430
+layout(location = 0) out vec4 color_out;
+void main() {
+  color_out = vec4(1.0, 0.0, 0.0, 1.0);
+}
+END
+
+SHADER vertex vert_shader_tex GLSL
+#version 430
+layout(location = 0) in vec4 position;
+layout(location = 1) in vec2 texcoords_in;
+layout(location = 0) out vec2 texcoords_out;
+void main() {
+  gl_Position = position;
+  texcoords_out = texcoords_in;
+}
+END
+
+SHADER fragment frag_shader_tex GLSL
+#version 430
+layout(location = 0) in vec2 texcoords_in;
+layout(location = 0) out vec4 color_out;
+uniform layout(set=0, binding=0) texture2D tex;
+uniform layout(set=0, binding=1) sampler tex_sampler;
+void main() {
+  color_out = texture(sampler2D(tex, tex_sampler), texcoords_in);
+}
+END
+
+BUFFER texture FORMAT R8G8B8A8_UNORM
+BUFFER framebuffer FORMAT B8G8R8A8_UNORM
+
+SAMPLER sampler_float_opaque_white ADDRESS_MODE_U clamp_to_border ADDRESS_MODE_V clamp_to_border BORDER_COLOR float_opaque_white
```

Comment:
Let's wrap these so they're easier to read:

```
SAMPLER sampler_float_opaque_white \
   ADDRESS_MODE_U clamp_to_border \
   ADDRESS_MODE_V clamp_to_border \
   BORDER_COLOR float_opaque_white
```

Reply:
Done